serverless/src/step_function/types.ts

```diff
@@ -0,0 +1,27 @@
+export interface StateMachine {
```

Comment:
These interfaces contains only the fields we need. Name and type are referred to AWS doc: https://docs.aws.amazon.com/AWSCloudFormation/latest/UserGuide/aws-resource-stepfunctions-statemachine.html